.eslintrc

```diff
@@ -16,3 +16,7 @@ globals:
   IteratorResult: true
   ReactClass: true
   ReactElement: true
+
+rules:
+  # allow use constructors for side-effects
+  no-new: 0
```

Comment:
Disabling like this won't work unless we do it internally at FB as well (since changes are still made internally first and synced out). And I'm not sure we want to disable it - it has value in real code (and it unfortunate we can't disable selectively in tests here). We could do this one on the per file basis perhaps…